main.py

```diff
@@ -6,109 +6,43 @@
     - measuring the respective contributions of each partner to the model performance (termed "contributivity")
 """
 
-import os
-
-import matplotlib.pyplot as plt
 from loguru import logger
 
-from mplc import scenario
 from mplc import utils
+from mplc import experiment
 from mplc.utils import parse_command_line_arguments
 
 DEFAULT_CONFIG_FILE = "./config.yml"
 
 
 @logger.catch
 def main():
-    args = parse_command_line_arguments()
 
-    if args.verbose:
-        debug = True
-    else:
-        debug = False
+    # Get arguments
+    args = parse_command_line_arguments()
 
+    # Initialize log level
+    debug = True if args.verbose else False
     utils.init_logger(debug)
-
     logger.debug("Standard output is sent to added handlers.")
 
-    if args.file:
-        logger.info(f"Using provided config file: {args.file}")
-        config = utils.get_config_from_file(args.file)
-    else:
-        logger.info(f"Using default config file: {DEFAULT_CONFIG_FILE}")
-        config = utils.get_config_from_file(DEFAULT_CONFIG_FILE)
-
-    scenario_params_list = utils.get_scenario_params_list(
-        config["scenario_params_list"])
-
-    experiment_path = config["experiment_path"]
-    n_repeats = config["n_repeats"]
-    validate_scenario_list(scenario_params_list, experiment_path)
-
-    for scenario_id, scenario_params in enumerate(scenario_params_list):
-        logger.info(f"Scenario {scenario_id + 1}/{len(scenario_params_list)}: {scenario_params}")
-
-    # Move log files to experiment folder
-    utils.set_log_file(experiment_path)
-
-    # GPU config
+    # Initialize GPU configuration
     utils.init_gpu_config()
 
-    # Close open figures
-    plt.close("all")
-
-    # Iterate over repeats of all scenarios experiments
-    for i in range(n_repeats):
-
-        logger.info(f"Repeat {i + 1}/{n_repeats}")
+    # Instantiate an Experiment
+    current_experiment = experiment.Experiment()
```

Comment:
`current` prefix is a bit weird as there is only one experiment. Would rename to `my_experiment` or something like that

Reply:
I renamed it `main_experiment`